lfx-mentorship/2023/01-Mar-May/project_ideas.md

```diff
@@ -23,9 +35,9 @@
 
 - Description: Cluster API Provider GCP (CAPG) enables the creation of Kubernetes clusters in GCP with Cluster API. With increasing adoption of Cluster API (CAPI) in general and of CAPG we want to improve the supportability of CAPG, especially for production environments. The first part of this is to add telemetry/tracing using OpenTelemetry so that we can understand and visualize the flow of reconciliation within the provider. The next part is to add a **pprof** endpoint that can be optionally enabled to enable operations/support users to collect profiling information from a running instances of CAPG.
 - Expected Outcome: This work will enable tracing and profiling of a running instance of CAPG (along with supporting docs) to supports operations/support engineers.
-Recommend Skills: Golang, Kubernetes
-Mentors(s): Carlos Panato (@cpanato), Richard Case (@richardcase)
-Upstream Issue: https://github.com/kubernetes-sigs/cluster-api-provider-gcp/issues/810
+- Recommend Skills: Golang, Kubernetes
+- Mentors(s): Carlos Panato (@cpanato), Richard Case (@richardcase)
+- Upstream Issue: https://github.com/kubernetes-sigs/cluster-api-provider-gcp/issues/810
```

Comment:
I'm not sure what this change is

Reply:
@xmulligan was being a good citizen; fixing the formatting issues in that page :)

Things look proper now: https://github.com/cncf/mentoring/blob/27b719d3bd3dc4695c585325f61e649d1cc9c2c1/lfx-mentorship/2023/01-Mar-May/project_ideas.md

/lgtm

Reply:
Yes, I was just trying to fix the formatting. I can break that out into a separate PR if you want

Reply:
Ah, thanks! The github review preview isn't clear.